src/middlewared/middlewared/plugins/zettarepl.py

```diff
@@ -554,6 +554,13 @@ async def get_definition(self):
                                           for periodic_snapshot_task in replication_task["periodic_snapshot_tasks"]]
             my_schedule = replication_task["schedule"]
 
+            properties_exclude = replication_task["properties_exclude"].copy()
+            properties_override = replication_task["properties_override"].copy()
+            for property in ["sharenfs", "sharesmb"]:
```

Comment:
Guess based on what you said in the other ticket we can also just add mountpoint here and call it a day?